src/frontend/test_runner/src/lib.rs

```diff
@@ -544,9 +537,19 @@ pub async fn run_test_file(file_name: &str, file_content: &str) {
     let cases: Vec<TestCase> = serde_yaml::from_str(file_content).unwrap();
     let cases = resolve_testcase_id(cases).expect("failed to resolve");
 
-    for c in cases {
+    for (i, c) in cases.into_iter().enumerate() {
+        println!(
+            "Running test #{i} (id: {}), SQL:\n{}",
+            c.id.clone().unwrap_or_else(|| "<none>".to_string()),
+            c.sql
+        );
         if let Err(e) = c.run(true).await {
-            println!("\nTest case failed, the input SQL:\n{}\n{}", c.sql, e);
+            eprintln!(
```

Comment:
Use stderr for failed instead of paniced test case to make it slightly clearer. For failed test case, stdout is redundant 🥸

<img width="1012" alt="image" src="https://user-images.githubusercontent.com/37948597/185996681-59cae3d7-9a67-4a87-8b06-6a5b2f54127b.png">